pages/collection/[id]/play/index.js

```diff
@@ -0,0 +1,265 @@
+  flex-grow: 1;
+  padding-bottom: 20px;
+`;
+
+const StyledFooter = styled.footer`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  padding: 30px;
+  z-index: 200;
+`;
+
+export default function PlayMode({ collections, flashCards }) {
+  // Router
+  const router = useRouter();
+  const { id, card } = router.query; // id for remaining in current collection, card for moving to next card/page
+
+  // States
+  const [showStopConfirm, setShowStopConfirm] = useState(true);
+  const [showFinalMessage, setShowFinalMessage] = useState(false);
+  const [showAnswer, setShowAnswer] = useState(false);
+  const [isCorrect, setIsCorrect] = useState(0);
+  const [timeElapsed, setTimeElapsed] = useState(0);
+  const [countDown, setCountDown] = useState(3);
+  const [isCounting, setIsCounting] = useState(true);
+
+  // Countdown Timer: Runs until 0, then shows quiz content
+  useEffect(() => {
+    if (countDown > 0) {
+      const countDownTimer = setInterval(() => {
+        setCountDown((prevTime) => prevTime - 1);
+      }, 1000);
+
+      return () => clearInterval(countDownTimer);
+    } else {
+      setIsCounting(false); // Countdown finished, show quiz
+    }
+  }, [countDown]);
+
+  //Timer
+
+  useEffect(() => {
+    if (showFinalMessage || !showStopConfirm) return; // Stop timer when quiz ends
+
+    const timer =
+      !isCounting &&
+      setInterval(() => {
+        setTimeElapsed((prevTime) => prevTime + 1);
+      }, 1000);
+
+    return () => clearInterval(timer); // Cleanup on unmount
+  }, [showFinalMessage, showStopConfirm, isCounting]);
+
+  // Format time as MM:SS
+  function formatTime(seconds) {
+    const minutes = Math.floor(seconds / 60);
+    const secs = seconds % 60;
+    return `${minutes}:${secs < 10 ? "0" : ""}${secs}`;
+  }
+
+  // Background Color
+  const currentCollection = collections.find(
+    (collection) => collection.id === id
+  );
+  if (!currentCollection) return <p>No collection found.</p>;
+
+  const backgroundColor = currentCollection.color;
+
+  // Pages / Cards
+  const filteredFlashCards = flashCards.filter(
+    (card) => card.collectionId === currentCollection.id
+  );
+  const totalPages = filteredFlashCards.length;
+  const currentPage = card ? parseInt(card, 10) : 0; // pareInt converts string into number, base 10 ensures number to be decimal
+
+  // Stop Functions
+  function handleToggle() {
+    setShowStopConfirm((prev) => !prev);
+  }
+  function handleConfirmStop() {
+    router.push(`/collection/${id}`);
+  }
+
+  // Nav Functions
+
+  function handleNext() {
+    if (currentPage < totalPages - 1) {
+      setShowAnswer(false);
+      // navigate to new URL while keeping current page structure
+      router.push({
+        pathname: router.pathname, // ensure to stay on same page
+        query: { id, card: currentPage + 1 }, // go to next card
+      });
+    } else {
+      setShowFinalMessage(true);
+    }
+  }
```

Comment:
don't use the navigation for changing the displayed card, use a state instead. you can then remove the push and set the card State when pressing next and prev.